libraries/AP_PiccoloCAN/piccolo_protocol/ECUDefines.c

```diff
@@ -0,0 +1,652 @@
+// ECUDefines.c was generated by ProtoGen version 3.2.a
+
```

Comment:
I think you've been adding a GPLv3 template in the past for this generated code?
you could also add a node in the license.txt in this directory saying "this license applies to all the files in this directory"